.fossa.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019-present Open Networking Foundation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
+# Visit https://fossa.com to learn more
+
+# Usage: FOSSA_API_KEY=<> fossa analyze
+# -T <team-name> is not supported at this moment from fossa side.
+
+version: 2
+cli:
+  server: https://app.fossa.com
+  fetcher: custom
+  project: openmme
+analyze:
+  modules:
+  - name: openmme
+    type: raw
+    target: ../openmme
```

Comment:
Does `.` not work?, or is a path required?

Reply:
Fossa technical person suggested to give a path during the zoom meeting when I show this file to them.